.github/actions/combine-results/action.yaml

```diff
@@ -0,0 +1,37 @@
+name: Combine results
+description: | 
+  Determines a single result from one or more results. A single skipped or
+  failed will cause the action to fail. Details for each job will be output 
+  as part of a results summary.
+
+inputs:
+  needs-json:
+    description: Job outputs in JSON format
+    required: true
+    type: string
+
+runs:
+  using: composite
+  steps:
+  - name: Summarise and combine results
```

Comment:
Perhaps given the latest announcement this should be with US spelling (summarize)?

Reply:
I think it this becomes a thing we probably want to do this across the project at once, rather than mixing things.